scripts/un/sdg/sdmx/m49.csv

```diff
@@ -0,0 +1,250 @@
+Country or Area	M49 code	ISO-alpha3 code
+Afghanistan	004	AFG
+Åland Islands	248	ALA
+Albania	008	ALB
+Algeria	012	DZA
```

Comment:
Just want to confirm the format of this file is correct. I see it's a .csv but not comma-delimited / not evenly spaced

Reply:
sorry changed to tsv